Gemfile

```diff
@@ -45,11 +45,26 @@ group :development, :test do
   gem 'rubocop'
   gem 'shoulda-context'
   gem 'shoulda-matchers'
+
+  #enable quick testing (in-memory db)
+  gem 'sqlite3'
```

Comment:
In the models PR we're using UUID-type columns for user and account IDs, which sqlite doesn't support. Do we really need to bring another db into the mix?